src/app/modules/dialog/components/start-service-dialog/start-service-dialog.component.ts

```diff
@@ -27,6 +28,8 @@ export interface StartServiceDialogResult {
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class StartServiceDialogComponent implements OnInit {
+  readonly requiredRoles = [Role.ServiceWrite];
```

Comment:
This check feels redundant. The ticket requires that this check is performed earlier. Meaning the user shouldn't even get to this dialog if the user doesn't have the required role.